packages/atomic/src/components/facets-v1/atomic-rating-range-facet/atomic-rating-range-facet.tsx

```diff
@@ -0,0 +1,242 @@
+import {Component, h, State, Prop, VNode, Host} from '@stencil/core';
+import {
+  NumericFacet,
+  buildNumericFacet,
+  NumericFacetState,
+  NumericFacetOptions,
+  SearchStatus,
+  SearchStatusState,
+  buildSearchStatus,
+  NumericFacetValue,
+  NumericRangeRequest,
+  buildNumericRange,
+} from '@coveo/headless';
+import {
+  Bindings,
+  BindStateToController,
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../utils/initialization-utils';
+import {FacetPlaceholder} from '../../facets/atomic-facet-placeholder/atomic-facet-placeholder';
+import {FacetContainer} from '../facet-container/facet-container';
+import {FacetHeader} from '../facet-header/facet-header';
+import {FacetValueLink} from '../facet-value-link/facet-value-link';
+import {FacetValueIconRating} from '../facet-value-icon-rating/facet-value-icon-rating';
+import {BaseFacet} from '../facet-common';
+import Star from '../../../images/star.svg';
+
+/**
+ * A facet is a list of values for a certain field occurring in the results, ordered using a configurable criteria (e.g., number of occurrences).
+ * An `atomic-rating-facet` displays a facet of the results for the current query as ratings.
+ * It only supports numeric fields.
+ *
+ * @part facet - The wrapper for the entire facet.
+ * @part placeholder - The placeholder shown before the first search is executed.
+ *
+ * @part label-button - The button that displays the label and toggles to expand or collapse the facet.
+ * @part label-button-icon - The label button icon.
+ * @part clear-button - The button that resets the actively selected facet values.
+ * @part clear-button-icon - The clear button icon.
+ *
+ * @part values - The facet values container.
+ * @part value-label - The facet value label, common for all displays.
+ * @part value-count - The facet value count, common for all displays.
+ *
+ * @part value-checkbox - The facet value checkbox, available when display is 'checkbox'.
+ * @part value-checkbox-label - The facet value checkbox clickable label, available when display is 'checkbox'.
+ * @part value-link - The facet value when display is 'link'.
+ * @part value-box - The facet value when display is 'box'.
+ *
+ */
+@Component({
+  tag: 'atomic-rating-range-facet',
+  styleUrl: '../atomic-rating-facet/atomic-rating-facet.pcss',
```

Comment:
It may be confusing in the future to change this other css file and for it to affect two components, since we may think of checking how it impacted this one.

I would prefer if either this file had a name that is different from that of both components in some way (e.g., by suffixing `-common`) and if both facets shared the same directory.

Reply:
That makes sense. Will do